pkgs/by-name/ou/outguess/package.nix

```
@@ -0,0 +1,36 @@
+{ fetchFromGitHub
+, stdenv
+, lib
+, autoconf
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "outguess";
+  version = "0.4";
+  src = fetchFromGitHub {
+    owner = "resurrecting-open-source-projects";
+    repo = "outguess";
+    rev = version;
+    hash = "sha256-yv01jquPTnVk9fd1tqAt1Lxis+ZHZqdG3NiTFxfoXAE=";
+  };
+
+  nativeBuildInputs = [ autoconf automake ];
+
+  buildPhase = ''
+    ./autogen.sh
+    ./configure --with-generic-jconfig --prefix=$out
+    make
+  '';
+
+  installPhase = ''
+    make install
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

Better to remove it.

---

Removed.